src/v1_0/rw/cfg.cpp

```diff
@@ -2,7 +2,7 @@
 
 namespace abb :: rws :: v1_0 :: rw :: cfg
 {
-    void loadCFGFile(RWSClient& client, const FileResource& resource)
+    std::string loadCFGFile(RWSClient& client, const FileResource& resource)
```

Comment:
Should not be a member function. `RWSClient` is not intended to have any business logic, it is just the communication layer.

Reply:
Sorry, my bad -- this function is not a member of any class, I don't know why I made this comment. It is ok.